resources/rules.k.tmp

```diff
@@ -296,7 +296,7 @@ rule ACCTCODE in SetItem( 1 )
     rule ((A -Int (X +Int C)) +Int (C -Int Y)) => (A -Int X) -Int Y
 
     rule #if C #then G -Int X #else G -Int Y #fi => G -Int (#if C #then X #else Y #fi)
-      requires #isVariable(G)
+      requires notBool #isConcrete(G)
```

Comment:
notBool isVariable(X) =/= isConcrete(G)
WTF, martin

Reply:
#isVariable is broken